syntax = "proto3";

message UserRequest {
  string name = 1;
}

message UserResponse {
  string name = 1;
  int32 age = 2;
}

service UserService {
  rpc GetUserInfo(UserRequest) returns (UserResponse);
}